toolchain/check/dump.cpp

```diff
@@ -38,6 +38,11 @@ LLVM_DUMP_METHOD static auto Dump(const Context& context, Parse::NodeId node_id)
   return Parse::Dump(context.parse_tree(), node_id);
 }
 
+LLVM_DUMP_METHOD static auto Dump(const Context& context, int32_t inst_id)
```

Comment:
Maybe?
```suggestion
LLVM_DUMP_METHOD static auto DumpInstId(const Context& context, int32_t inst_id)
```

Reply:
I would feel more comfortable with this if this was first discussed in Discord, so it was clearer if this was what the people doing debugging want. I could easily imagine different people debugging different systems wanting a different different id by default.